about.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Unique Computers</title>
+</head>
+<body>
+    <div id="container">
+        <section>
+            <div id="About">
+                <img src="" alt="About Us">
+            </div>
+        </section>
+        <section>
+            <div>
+                <h2>About Us</h2>
+                <p>
+                    Unique Computers is a Nigerian computer company with its headquarters in Ikeja, Lagos and is 
+                    one of the largest Nigerian IT manufacturing companies in the world. The company develops 
+                    personal computers and other products related to these types of computers, builds and sells. The 
+                    Unique computers main office is located at Allen Avenue and has more than 180 employees in Nigeria.
+                </p>
```

Comment:
We need more information. Add more information about us. e.g What we do, Our products and Services etc